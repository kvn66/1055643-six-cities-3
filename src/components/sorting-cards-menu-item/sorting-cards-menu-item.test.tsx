import * as React from 'react';
import * as renderer from 'react-test-renderer';
import SortingCardsMenuItem from "./sorting-cards-menu-item";
import {ReactTestRendererJSON} from "react-test-renderer";

const testFn: jest.Mock = jest.fn();

it(`Render SortingCardsMenuItem`, () => {
  const tree: ReactTestRendererJSON = renderer
    .create(
        <SortingCardsMenuItem
          sortingMethod={0}
          selectedSortingMethod={0}
          tabIndex={0}
          setSortingMethod={testFn}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
