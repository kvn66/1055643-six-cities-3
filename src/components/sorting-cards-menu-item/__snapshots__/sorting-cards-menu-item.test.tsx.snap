// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SortingCardsMenuItem 1`] = `
<li
  className="places__option places__option--active"
  onClick={[Function]}
  tabIndex={0}
>
  Popular
</li>
`;
